css/style.css

```diff
@@ -0,0 +1,291 @@
+* {
+   box-sizing: border-box;
+}
+
+.body {
+   max-width: 1440px;
+   margin: 0 auto;
+   font-family: "Montserrat", sans-serif;
+   color: #000;
+ }
+
+ .header {
+   margin: 1.7% 3.8% 0 2.8%;
+   display: flex;
+   justify-content: space-between;
+   align-items: center;
+ }
+
+ .header__img {
+   display: block;
+ }
+
+ .header__menu_link {
+   display: block;
+   color: #000;
+   margin-left: 3%;
+   font-style: normal;
+   font-weight: 600;
+   font-size: 15px;
+   text-decoration: none;
+   line-height: 18px;
+   transition: all .2s;
+   transition-property: color, text-decoration;
+ }
+
+ .header__menu_link:hover {
+   color: #e51937;
+   text-decoration: underline;
+ }
+
+ .header__menu {
+   display: flex;
+   justify-content: space-around;
+ }
+
+ .header__menu_link-red {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)